src/experiments/logins/schema.json

```diff
@@ -17,12 +17,23 @@
           "optional": false,
```

Comment:
This looks great! I think we need to make the hostname optional here in the schema, otherwise LGTM

Reply:
that's fair; it is enforced lower down anyway, which is for the best.